src/shared/use-server-time-as-local/use-server-date-time-as-local.js

```diff
@@ -0,0 +1,21 @@
+import { useConfig } from '@dhis2/app-runtime'
+
+export const useServerDateTimeAsLocal = (dateTime) => {
+    const { systemInfo } = useConfig()
+
+    if (!dateTime) {
+        return undefined
+    }
+
+    const localNow = new Date()
+    localNow.setMilliseconds(0)
+    const nowAtServerTimeZone = new Date(
+        localNow.toLocaleString('en-US', {
```

Comment:
For practical purposes, I think this is okay, but the date parsing is implemented by browser and is not guaranteed to work on non-ISO formats (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse). I think we can just live with this as it seems major browsers parse dates well, but we could also select a locale that is closer to ISO format (e.g. 'sv-SE') if we wanted

Reply:
I think you may be overly cautious here. The MDN docs for `Date.prototype.toLocaleString()` mention in the [parameters section](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString#parameters) that a _BCP 47 language tag_ may be provided as a locale string.

Reply:
Yeah...I think that it doesn't matter honestly 👍. I think MDN also just notes that the Date() instantiation can be implementation-specific, but I think all major browsers in reality parse correctly from localeDateStrings, so I don't think it's worth it to code against the possibility that browsers start making their parse implementation strange.